.github/workflows/test.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Install App::Prove6
         run: zef install --verbose --/test App::Prove6
       - name: Run Tests
-        run: prove6 -I. t
+        run: raku -I.,lib t/01-basic.rakutest
```

Comment:
Test comment: Never seen this before. How does it work?

Reply:
:) sorry for the delay - I've been away for a few days. `prove6` and friends grab and hide output, but running `raku` itself to do the test emits more detail for debug. I swapped to this so that I could track down a test failure - turns out that in the VM the `.precomp` folders (i.e. home) are in the cwd.